### DISCUSSION TITLE
Filmon plugin Ubuntu error

### DISCUSSION THREAD

**Discussion by paul0008** [streamlink/streamlink#5432] [Q&A] [answered]
### Description

This is Filmon plugin and Ubuntu specific error. There isn't an eror in Windows.
It doesn't matter how I install streamlink, and which version od streamlink, and which version of Ubuntu, error is always there.
I would like to ask if someone knows a workaround for this errors.

This is streamlink version I run for this test, and how I installed it:

pipx install ./streamlink-5.5.1.tar.gz

Errors:

Ubuntu 22.04.02 LTS :
`
streamlink https://www.filmon.com/tv/cisco-kid-tv best
[cli][info] Found matching plugin filmon for URL https://www.filmon.com/tv/cisco-kid-tv
error: Unable to open URL: https://www.filmon.com/tv/cisco-kid-tv (HTTPSConnectionPool(host='www.filmon.com', port=443): Max retries exceeded with url: /tv/cisco-kid-tv (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:997)'))))
`
Ubuntu 23.04 :
`
streamlink --loglevel debug https://www.filmon.com/tv/cisco-kid-tv best
[cli][debug] OS:         Linux-6.2.0-24-generic-x86_64-with-glibc2.37
[cli][debug] Python:     3.11.2
[cli][debug] Streamlink: 5.5.1
[cli][debug] Dependencies:
[cli][debug]  certifi: 2023.5.7
[cli][debug]  isodate: 0.6.1
[cli][debug]  lxml: 4.9.3
[cli][debug]  pycountry: 22.3.5
[cli][debug]  pycryptodome: 3.18.0
[cli][debug]  PySocks: 1.7.1
[cli][debug]  requests: 2.31.0
[cli][debug]  urllib3: 2.0.3
[cli][debug]  websocket-client: 1.6.1
[cli][debug] Arguments:
[cli][debug]  url=https://www.filmon.com/tv/cisco-kid-tv
[cli][debug]  stream=['best']
[cli][debug]  --loglevel=debug
[cli][info] Found matching plugin filmon for URL https://www.filmon.com/tv/cisco-kid-tv
error: Unable to open URL: https://www.filmon.com/tv/cisco-kid-tv (HTTPSConnectionPool(host='www.filmon.com', port=443): Max retries exceeded with url: /tv/cisco-kid-tv (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:992)'))))
`
Ubuntu 23.04, with --http-no-ssl-verify :
`
streamlink --loglevel debug --http-no-ssl-verify https://www.filmon.com/tv/cisco-kid-tv best
[cli][debug] OS:         Linux-6.2.0-24-generic-x86_64-with-glibc2.37
[cli][debug] Python:     3.11.2
[cli][debug] Streamlink: 5.5.1
[cli][debug] Dependencies:
[cli][debug]  certifi: 2023.5.7
[cli][debug]  isodate: 0.6.1
[cli][debug]  lxml: 4.9.3
[cli][debug]  pycountry: 22.3.5
[cli][debug]  pycryptodome: 3.18.0
[cli][debug]  PySocks: 1.7.1
[cli][debug]  requests: 2.31.0
[cli][debug]  urllib3: 2.0.3
[cli][debug]  websocket-client: 1.6.1
[cli][debug] Arguments:
[cli][debug]  url=https://www.filmon.com/tv/cisco-kid-tv
[cli][debug]  stream=['best']
[cli][debug]  --loglevel=debug
[cli][debug]  --http-no-ssl-verify=True
[cli][info] Found matching plugin filmon for URL https://www.filmon.com/tv/cisco-kid-tv
error: Unable to open URL: https://www.filmon.com/tv/cisco-kid-tv (Cannot set verify_mode to CERT_NONE when check_hostname is enabled.)
`

**Comment by bastimeyer** [comment, marked as answer]
#5425
